package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "ember-cli-version-checker": "^1.2.0",
-    "execa": "^0.7.0",
+    "execa": "^0.7.0 || ^0.8.0 || ^0.9.0",
```

Comment:
Thanks, I didn't know that one before. And as `0.7.0 || <1.0.0` would still allow versions smaller than `0.7.0`, I assume `>=0.7.0 <1.0.0` would work best here?